packages/dds/sequence/src/test/intervalRevertibles.fuzz.spec.ts

```diff
@@ -0,0 +1,239 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import * as path from "path";
+import { readdirSync } from "fs";
+import { strict as assert } from "assert";
+import {
+	createWeightedAsyncGenerator as createWeightedGenerator,
+	AsyncGenerator as Generator,
+	takeAsync as take,
+} from "@fluid-internal/stochastic-test-utils";
+import {
+	createDDSFuzzSuite,
+	DDSFuzzModel,
+	DDSFuzzHarnessEvents,
+	DDSFuzzSuiteOptions,
+} from "@fluid-internal/test-dds-utils";
+import { TypedEventEmitter } from "@fluidframework/common-utils";
+import {
+	IFluidDataStoreRuntime,
+	IChannelServices,
+	IChannelAttributes,
+} from "@fluidframework/datastore-definitions";
+import { SharedStringFactory } from "../sequenceFactory";
+import {
+	appendAddIntervalToRevertibles,
+	// appendChangeIntervalToRevertibles,
+	// appendDeleteIntervalToRevertibles,
+	// appendIntervalPropertyChangedToRevertibles,
+	// appendSharedStringDeltaToRevertibles,
+} from "../revertibles";
+import { SharedString } from "../sharedString";
+import { assertEquivalentSharedStrings } from "./intervalUtils";
+import {
+	Operation,
+	FuzzTestState,
+	RevertOperation,
+	makeReducer,
+	RevertibleSharedString,
+	isRevertibleSharedString,
+} from "./intervalCollection.fuzzUtils";
+import { makeOperationGenerator } from "./intervalCollection.fuzz.spec";
+import { minimizeTestFromFailureFile } from "./intervalCollection.fuzzMinimization";
+
+// Note: none of these options are currently exercised, since the fuzz test fails with pretty much
+// any configuration due to known bugs. Once shared interval collections are in a better state these
+// should be revisited.
+interface OperationGenerationConfig {
```

Comment:
you shouldn't need to redefine this or the default options object; i'd prefer exporting them from `intervalCollection.fuzz.spec` (or even better, moving them to `intervalCollection.fuzzUtils` and importing from there in both fuzz test files)